usage/dist/licensing/README.md

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+This uses a special maven plugin to create license and notification information,
+based on metadata in overrides.yaml and extras.yaml.
+
+First install  https://github.com/ahgittin/license-audit-maven-plugin 
```

Comment:
Any chance of getting this into Maven Central? I'd prefer to avoid instructions like "download this and install it, then..." (although since this plugin is only loosely bound into the project, it's not a big issue.)